.travis.yml

```
@@ -10,14 +10,14 @@ env:
 jobs:
   fast_finish: true
   include:
-    - php: 7.1
+    - php: 7.4
```

We still need to test PHP 7.3 for now.

```suggestion
    - php: 7.3
```

---

Done